### ISSUE TITLE
installing and building fastq-join on a linux server.

### ISSUE THREAD

**Issue by tbranck28** [ExpressionAnalysis/ea-utils#67]
Hello, 

I am new to the command line and would appreciate help getting started with fastq-join. Can anyone give direction for how to install and build ea-utils on a linux server once the files have been downloaded and unzipped?

Thank you!

**Comment by drsuuzzz** [comment]
You should be able to just type _make_ at the command line and the executables will be built for you, https://github.com/ExpressionAnalysis/ea-utils/blob/wiki/Compiling.md

**Comment by tbranck28** [comment]
Thank you for the quick response. I am in the ExpressionAnalysis-ea-utils-bd148d4 directory and typed in make but am given this error:

make: *** No targets specified and no makefile found.  Stop.

**Comment by drsuuzzz** [comment]
cd into the clipper directory and you should see the Makefile there.  Try the _make_ command in that directory

**Comment by tbipf** [comment]
Ok thank you! new error. We have limited permission to download/install programs sometimes without our IT's permission. Does the following error look like that is the issue or is it something I am doing wrong?

cd sparsehash-2.0.3; ./configure; make
/bin/sh: ./configure: Permission denied
make[1]: Entering directory `/home/tobyn.branck/ExpressionAnalysis-ea-utils-bd148d4/clipper/sparsehash-2.0.3'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory `/home/tobyn.branck/ExpressionAnalysis-ea-utils-bd148d4/clipper/sparsehash-2.0.3'
make: *** [sparsehash] Error 2

**Comment by drsuuzzz** [comment]
Yeah, that seems like a configuration problem.  Sorry, seems like you can't avoid IT

**Comment by tbranck28** [comment]
I was worried about that - thanks for your help!

**Comment by MonicaSteffi** [comment]
how did you resolve it
